core/src/edu/rice/comp413fall2020grey/Location/locator/LocatorMain.java

```diff
@@ -0,0 +1,11 @@
+package edu.rice.comp413fall2020grey.Location.locator;
+
+import org.bson.conversions.Bson;
+
+public interface LocatorMain {
+    /*
+     * Method called by the Object Storage to execute new Area-of-Interest.
+     */
+    void updateInterest(Bson predicate);
```

Comment:
Are we still doing this or did we agree we would propagate updates through replication?